lib/jsdom/living/xmlhttprequest.js

```diff
@@ -585,6 +585,8 @@ module.exports = function createXMLHttpRequest(window) {
             }
           } else if (body instanceof ArrayBuffer) {
             flag.body = new Buffer(new Uint8Array(body));
+          } else if (ArrayBuffer.isView(body)) {
+            flag.body = new Buffer(body.buffer);
```

Comment:
Good catch! @domenic, do you think I shall add a test for this particular feature to WPT as well?